### ISSUE TITLE
can not save images to local

### ISSUE THREAD

**Issue by TrumanXia** [inhumantsar/slurp#60]
hello, any plan adding this feature?

**Comment by RobertBernstein** [comment]
I'd love to have this capability, as well.

**Comment by inhumantsar** [comment]
I am planning to add this feature (see #45) but haven't had time to work on slurp lately. 

there's been a couple of personal things which have taken more of my time than expected, though they are nearly out of the way.

tldr: Soon™️

**Comment by RobertBernstein** [comment]
Thanks!  Good luck with your personal things. 😊

**Comment by TrumanXia** [comment]
> I am planning to add this feature (see #45) but haven't had time to work on slurp lately.
> 
> there's been a couple of personal things which have taken more of my time than expected, though they are nearly out of the way.
> 
> tldr: Soon™️

glad to hear that! thank you!
